uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Service/CiviHRStatsGathererTest.php

```diff
@@ -214,20 +213,56 @@ public function testContactSubtypeCountWillMatchExpectedCount() {
   }
 
   public function testDeletedEntitiesWillNotBeIncluded() {
+    $contactID = ContactFabricator::fabricate()['id'];
     $existingContactCount = $this->getCurrentContactCount();
-    $deletedContact = ContactFabricator::fabricate();
-    $deletedContactID = $deletedContact['id'];
-    civicrm_api3('Contact', 'delete', ['id' => $deletedContactID]);
-    $params = ['is_deleted' => 1, 'id' => $deletedContactID];
-    $stillExists = civicrm_api3('Contact', 'getcount', $params) == 1;
-
-    if (!$stillExists) {
-      $this->markTestSkipped('There was an error in soft deletion of contact');
-    }
+    TaskTypeFabricator::fabricate();
+    $this->setUpLeaveRequest($contactID);
+    $this->registerCurrentLoggedInContactInSession($contactID);
+    civicrm_api3(
+      'UFMatch',
+      'delete',
+      ['id' => UFMatchFabricator::fabricate()['id']]
+    );
```

Comment:
This method ended up being quite long. What about something like this

```php
civicrm_api3('UFMatch', 'delete', [ 'id' => UFMatchFabricator::fabricate()['id'] ]);
```
Or like this, if you think the line is too long:

```php
$ufMatch = UFMatchFabricator::fabricate();
civicrm_api3('UFMatch', 'delete', ['id' => $ufMatch['id']]);
```

Also, to improve readability, please blank lines between groups of statements